engine/server/server.go

```diff
@@ -101,6 +112,28 @@ func startJob(model string, out string, jobs map[string]*os.Process) (string, er
 	return id, err
 }
 
+func xmlConvert(model string, out string, params *pb.Params) (string){
```

Comment:
change return type to `(string, error)` here and instead of `panic` below, just return the error